src/v3/src/types/widget.ts

```diff
@@ -79,6 +80,8 @@ export type WidgetOptions = {
 
   // theming
   theme?: ThemeOptions;
+  // Override MUI Theming
+  muiThemeOptions?: MuiThemeOptions;
```

Comment:
Not entirely sure what to name this since there is already a `theme` property. Also, not sure if I should have included it inside the existing `theme` property. Looking for feedback on these.

Reply:
👏 I'll defer to developers!

Separate properties for different kinds of theming is probably helpful clarity?

On one hand, I'd prefer MUI to be an implementation detail that customers don't have to worry about. But on the other hand, if the API is defined by MUI and we'll be sending them to MUI docs for reference, having MUI in the name is probably helpful clarity?

Reply:
Exactly this, that is where I struggled because I wanted to keep MUI abstracted, however, since we don't control the properties/API that is needed to be passed in, I figured it would make more sense to have `mui` in the name so the enduser can make the correlation. 

Let's see what others think, I am torn with this as well 😅

Reply:
I think leaving `mui` makes sense, since in some place we will likely need to doc that it's MUI underlying anyways.

Any thoughts on using `overrides` rather than `options`?

Reply:
I actually prefer `overrides` lol (I used it as the param name to the function) not sure why I put options here 😅